bag_of_words/code/bag_of_words.py

```diff
@@ -0,0 +1,36 @@
+# Read papers and convert them to bag-of-words representations.
+
+indir = "../data/raw/pubmed-txts-mf/"
+outdir = "../data/processed/"
+from sklearn.feature_extraction.text import CountVectorizer
+vectorizer = CountVectorizer()
+import os
+import numpy as np
+import pickle
+
+# Read papers into a list
+pmids = os.listdir(indir)
+pmids.sort()
+# pmids = pmids[:5]  # For debugging
+corpus = [''] * len(pmids)

```

Comment:
Since we end up iterating over the PMID to build the corpus, and indexing by the PMID, I recommend using a dict of strings here instead of a list. That way, you can refer to the PMID as the dict key, and you don't need to keep a separate list of PMID.